windows/win_feature.ps1

```diff
@@ -47,40 +47,52 @@ Elseif (!$params.state) {
 If ($params.restart) {
     $restart = $params.restart | ConvertTo-Bool
 }
-Else
-{
+Else {
     $restart = $false
 }
 
-if ($params.include_sub_features)
-{
+If ($params.include_sub_features) {
     $includesubfeatures = $params.include_sub_features | ConvertTo-Bool
 }
-Else
-{
+Else {
     $includesubfeatures = $false
 }
 
-if ($params.include_management_tools)
-{
+If ($params.include_management_tools) {
     $includemanagementtools = $params.include_management_tools | ConvertTo-Bool
 }
-Else
-{
+Else {
     $includemanagementtools = $false
 }
 
+If ($params.source) {
+    $source = $params.source.ToString().ToLower()
+    If (!Test-Path $source) {
```

Comment:
I'm getting an execution error from "!test-path". You probably need to specify it as if (!(test-path $source)) {